File: lab-3-ddb/environment/python_3.6.8/create_table.py

import <FMI>
DDB_RESOURCE = boto3.resource("dynamodb", region_name="<FMI>")

DDB_RESOURCE.<FMI>(
    TableName="<FMI>",
    KeySchema=[
        {
            "AttributeName": "petname",
            "KeyType": "<FMI>"
        }
    ],
    AttributeDefinitions=[
        {
            "AttributeName": "petname",
            "AttributeType": "<FMI>"
        }
    ],
    ProvisionedThroughput={
        "ReadCapacityUnits": <FMI>,
        "WriteCapacityUnits": 1
    }
)

File: lab-3-ddb/environment/ruby_2.6.0/solution/create_table.rb
require "aws-sdk-dynamodb"
ddb_client = Aws::DynamoDB::Client.new(region: "us-east-1")

resp = ddb_client.create_table({
  attribute_definitions: [
    {
      attribute_name: "petname", 
      attribute_type: "S", 
    }
  ], 
  key_schema: [
    {
      attribute_name: "petname", 
      key_type: "HASH", 
    }
  ], 
  provisioned_throughput: {
    read_capacity_units: 1, 
    write_capacity_units: 1, 
  }, 
  table_name: "lostcats", 
})

puts resp.to_h
